### DISCUSSION TITLE
Adding wordwise and xray to existing files

### DISCUSSION THREAD

**Discussion by catpool91** [xxyzz/WordDumb#177] [Q&A]
What is the way to follow if I already have epubs that I sent with calibre and I want to add wordwise and xray without losing the annotations made. Thank you!

**Comment by xxyzz** [comment]
I assume you're using Kindle? Sorry I don't know how annotations are stored on Kindle.  I could only recommend you don't send EPUB to Kindle but convert the book to a Kindle format then use the plugin to upload the book and Word Wise files to Kindle.
